Makefile

```diff
@@ -1,15 +1,26 @@
 BIN_OUTPUT_PATH = bin
 TOOL_BIN = bin/gotools/$(shell uname -s)-$(shell uname -m)
+ARM64_OUTPUT = $(BIN_OUTPUT_PATH)/raspberry-pi/arm64
+ARM32_OUTPUT = $(BIN_OUTPUT_PATH)/raspberry-pi/arm32
+DOCKER_ARCH ?= arm64
 
 .PHONY: module
 module: build
 	rm -f $(BIN_OUTPUT_PATH)/raspberry-pi-module.tar.gz
 	tar czf $(BIN_OUTPUT_PATH)/raspberry-pi-module.tar.gz $(BIN_OUTPUT_PATH)/raspberry-pi run.sh meta.json
 
 .PHONY: build
-build:
-	rm -f $(BIN_OUTPUT_PATH)/raspberry-pi
-	go build -o $(BIN_OUTPUT_PATH)/raspberry-pi main.go
+build: build-arm64
+
+.PHONY: build-arm64
+build-arm64:
+	rm -f $(ARM64_OUTPUT)
+	GOARCH=arm64 go build -o $(ARM64_OUTPUT) main.go
+
+.PHONY: build-arm32
+build-arm32:
+	rm -f $(ARM32_OUTPUT)
+	GOARCH=arm GOARM=7 go build -o $(ARM32_OUTPUT) main.go
```

Comment:
I'd suggest simplifying this with a single, variable build... see here for how I do the same in Agent:

https://github.com/viamrobotics/agent/blob/main/Makefile#L23-L38

Only the last target is the real build (and it's more complicated than yours, so you can ignore most of it.) Mostly pointing out how to alias the other architectures simply. Having the same output file name also simplifies other stuff in workflows and the like where you can just change a single variable (or the platform itself) and not have to juggle lots of variable filenames and such.